core/src/components/textarea/textarea.ionic.scss

```diff
@@ -6,6 +6,7 @@
 // --------------------------------------------------
 
 :host {
+  --border-color: #{globals.$ion-primitives-neutral-500};
```

Comment:
```suggestion

```

This shouldn't need to be added because we have it in `textarea.ionic.outline.scss`